rocAL/include/parameters/parameter_random.h

```diff
@@ -81,8 +108,10 @@ class UniformRand : public Parameter<T> {
     T _start;
     T _end;
     T _updated_val;
+    std::vector<T> _elements;
```

Comment:
Can we have a different name?
Also please add comment as to what this variable contain

Reply:
Address this.

Reply:
Renamed _elements to _values